src/views/mainLayout.js

```diff
@@ -14,6 +14,27 @@ const mainLayout = data => {
   const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
   const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
 
+  const shouldShowRecruitmentBanner = () => {
+    // Only show if ENVs are set and user language is set to "en"
+    if (
+      AppConstants.RECRUITMENT_BANNER_TEXT &&
+      AppConstants.RECRUITMENT_BANNER_LINK &&
+      getLocale().includes('en')
+    ) {
+      return true
+    }
+    return false
+  }
+
+  const recruitmentBanner = () => ` 
+    <div role="banner" class="recruitment-banner" hidden>
+      <a id="recruitment-banner" class="text-link" href="${AppConstants.RECRUITMENT_BANNER_LINK}" target="_blank" rel="noopener noreferrer" data-ga="send-ga-pings" data-event-category="Recruitment" data-event-label="${AppConstants.RECRUITMENT_BANNER_TEXT}">${AppConstants.RECRUITMENT_BANNER_TEXT}</a>
```

Comment:
Not sure if it's confusing to have a `<div class="recruitment-banner">` and nested `<a id="recruitment-banner">` or if one of those names should change to be less ambiguous (since I was originally a bit confused by the CSS's `#recruitment-banner`-vs-`.recruitment-banner`).

`recruitment-banner-wrapper`?
`recruitment-banner-link`?

Reply:
Actual answer: This was some expert copy pasta [from Relay](https://github.com/mozilla/fx-private-relay/blob/v3.1.20/privaterelay/templates/includes/header.html#L14-L20) (which was actually copy pasta [from Monitor](https://github.com/mozilla/blurts-server/blob/5ef498847ad44cf3c7e61c001a8a42cc88d03215/template-helpers/hbs-helpers.js#L56-L61)) ♻️ 

Regardless - I'll rename the markup (and CSS) to be more logical.

Reply:
I updated the ID names to be more logical. We now have: 

```HTML
<div class="recruitment-banner" hidden>
      <a id="recruitment-banner-link" >...</a>
      <button id="recruitment-banner-dimiss">
            <svg/>...</svg>
      </button>
</div>
```

Revised in ad11567d72b884d0a90d9cd5b9a365d29d1569cb